test cases/common/234 very long commmand line/meson.build

```diff
@@ -6,6 +6,9 @@ if build_machine.system() == 'windows'
   # cmd.exe: 8kb
   # CreateProcess: 32kb
   limit = 32767
+  # NOTE: filename limit is 260 characters unless
```

Comment:
If I've understood correctly what this change is doing, this comment should probably mention that the ninja backend will use absolute filenames, so we go through these gynmastics to arrange that they don't exceed that limit.